### PR TITLE
Make cli log message clear, reduce noise "boxen" style, fix #560

### LINKED ISSUE #560: fix formatting of summary in cli workflow
## Affected Packages

`pkg:cli`

## Problem

The summarization of a changeset in the CLI is a nice idea, but right now the formatting seems broken:

image

## Proposed solution

1. I think if you open the editor to write the changeset (instead of doing it directly in the terminal) the summary step should be eliminated. Because you've already just seen the entire changeset file and have saved and closed it.

2. The formatting should be fixed and simplified. Potentially something like this:

```
🦋  === Summary of changeset ===
🦋  minor: slate, slate-react
🦋  patch: slate-history
🦋  
🦋  Note: all dependents of these packages that are incompatible with the new version 
🦋  will be patch bumped when this changeset is applied.
🦋  
🦋  Is this your desired changeset? (Y/n) · true
```

Which includes things like:

- Reduce noise around the "note", using just simply red+bold on the word "Note:" will get the job done, and right now it's so noisey that it makes other pieces hard to read.
- Reduce noise on `major/minor/patch`. Again color will get the job done. (And the formatting I've used mimics how they are formatted in the questions already, which is good.)
- Collapse packages onto a single line since the majority case is have either all in one type (and a long list), or just a few packages changing, so no need to make it verbose.
- Collapse header into a single line, since it was unnecessary to have two before.

### LINKED ISSUE #684: audit warning for outdated ansi-regex subdependency of boxen@1 used by @changesets/cli
## Affected Packages

cli

## Problem

running `pnpm audit` reports a possible security issue with an old version of ansi-regex used by boxen@1.

pnpm audit
```
┌─────────────────────┬────────────────────────────────────────────────────────────────┐
│ moderate            │  Inefficient Regular Expression Complexity in chalk/ansi-regex │
├─────────────────────┼────────────────────────────────────────────────────────────────┤
│ Package             │ ansi-regex                                                     │
├─────────────────────┼────────────────────────────────────────────────────────────────┤
│ Vulnerable versions │ >2.1.1 <5.0.1                                                  │
├─────────────────────┼────────────────────────────────────────────────────────────────┤
│ Patched versions    │ >=5.0.1                                                        │
├─────────────────────┼────────────────────────────────────────────────────────────────┤
│ More info           │ https://github.com/advisories/GHSA-93q8-gq69-wqmw              │
└─────────────────────┴────────────────────────────────────────────────────────────────┘
```
pnpm why ansi-regex
```
@changesets/cli 2.18.1
├─┬ boxen 1.3.0
│ ├─┬ ansi-align 2.0.0
│ │ └─┬ string-width 2.1.1
│ │   └─┬ strip-ansi 4.0.0
│ │     └── ansi-regex 3.0.0
```

## Proposed solution

Given that the latest major version of boxen is 6, please consider upgrading it.

### PR DESCRIPTION
**Pull request by zthxxx**
and also unify visual style of text font.

## Affected Packages

`@changeset/cli`

## Problem

Summary note is too noisy, closes #560. 

(and this also fixes #684)

### After

image

### Before

image

### CODE REVIEW

**Comment by zthxxx** [comment]
cc: @ianstormtaylor @Andarist  @mitchellhamilton

File: packages/cli/package.json [hunk 1]
```diff
@@ -38,7 +38,6 @@
     "@changesets/write": "^0.1.6",
     "@manypkg/get-packages": "^1.1.3",
     "@types/semver": "^6.0.0",
-    "boxen": "^1.3.0",
```
Comment (webmaster128) [inline packages/cli/package.json:41]: 😍

**Comment by zthxxx** [comment]
@Andarist require approve

**Comment by Andarist** [comment]
@zthxxx thank you for the PR - I hope to review it soon-ish but atm I have some other, more pressing, issues to handle. Please don't think that your work is not appreciated - it's just waiting in the queue (well, more like a pile) of things to handle on my side.

**Comment by zthxxx** [comment]
@Andarist  Do you have a minute? I think this PR is small and just some format of logging with 4 sentences.

Maybe it will only takes about 5min to review.

File: .changeset/gorgeous-glasses-retire.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@changesets/cli": patch
+---
+
+Make cli log message clear, reduce noise "boxen" style, unify visual style of text font, discussion in #560
```
Comment (Andarist) [inline .changeset/gorgeous-glasses-retire.md:5]:
```suggestion
Reworked the CLI output when adding a changeset. This includes fixing broken formatting in certain terminal width, reducing visual noise and making other small adjustments.
```

File: .changeset/gorgeous-glasses-retire.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@changesets/cli": patch
```
Comment (Andarist) [inline .changeset/gorgeous-glasses-retire.md:2]:
```suggestion
"@changesets/cli": minor
```

**Comment by zthxxx** [comment]
thanks for your review

**Pull request merged**